cryptofeed/bitstamp/pairs.py

```diff
@@ -16,5 +18,8 @@ def gen_pairs():
         ret[normalized] = pair
     return ret
 
-
-bitstamp_pair_mapping = gen_pairs()
+try:
+    bitstamp_pair_mapping = gen_pairs()
+except:
+    LOG.exception("Unable to get pairs from Bitstamp")
+    bitstamp_pair_mapping = None
```

Comment:
dont set to none, set to a hard coded list of pairs, see https://github.com/bmoscon/cryptofeed/blob/master/cryptofeed/gemini/pairs.py as an example